iotedgedev/modules.py

```diff
@@ -239,6 +232,25 @@ def build_push(self, template_file, default_platform, no_build=False, no_push=Fa
 
         return gen_deployment_manifest_path
 
+    def _update_module_maps(self, placeholder_base, module_name, module, placeholder_tag_map, tag_build_profile_map, default_platform):
+        try:
+            for platform in module.platforms:
+                # get the Dockerfile from module.json
+                dockerfile = module.get_dockerfile_by_platform(platform)
+                container_tag = "" if self.envvars.CONTAINER_TAG == "" else "-" + self.envvars.CONTAINER_TAG
+                tag = "{0}:{1}{2}-{3}".format(module.repository.lower(), module.tag_version, container_tag, platform)
+                placeholder_tag_map["${{{0}.{1}}}".format(placeholder_base, platform)] = tag
+                placeholder_tag_map[tag] = tag
+
+                if platform == default_platform:
+                    placeholder_tag_map["${{{0}}}".format(placeholder_base)] = tag
+                elif platform == default_platform + ".debug":
+                    placeholder_tag_map["${{{0}.{1}}}".format(placeholder_base, "debug")] = tag
+
+                tag_build_profile_map[tag] = BuildProfile(module_name, dockerfile, module.context_path, module.build_options)
```

Comment:
module_name here actually is not necessary. It seems only used to output some log later.

Reply:
Yes, the logs will help users understand what's being built so we could keep it here.